openstudiocore/src/utilities/units/Scale.cpp

```diff
@@ -114,6 +114,20 @@ namespace openstudio{
     static Scale M = {"M", "mega",6,1.0E6};
     return M;
   }
+  
+  /** Static constant defining SI prefix for 10^5. */
+  UTILITIES_API const Scale& hectokilo()
+  {
+    static Scale hk = {"_hk", "hectokilo",5,1.0E5};
+    return hk;
+  }
+
+  /** Static constant defining SI prefix for 10^4. */
```

Comment:
@macumber I think you should dump myria.  First, because it apparently uses "k", which is confusing when we have "kilo" and second because it is [obsolete](https://en.wikipedia.org/wiki/Myria-)